ctapipe/calib/camera/mc.py

```diff
@@ -0,0 +1,578 @@
+        sum_pix = []
+        peak_pix = []
+        for ipix in range(0, get_num_pixels(telid)):
+            i = 0
+            knb = 0
+            # Loop over the neighbors of ipix
+            ipix_nb = geom.neighbors[ipix]
+            nb_samples = [0 for ii in range(get_num_samples(telid))]
+            for inb in range(len(ipix_nb)):
+                nb_samples += np.array(get_adc_sample(telid, igain)
+                                       [ipix_nb[inb]])
+                knb += 1
+            if parameters['lwt'] > 0:
+                for isamp in range(1, get_num_samples(telid)):
+                    nb_samples += np.array(get_adc_sample(telid, igain)
+                                           [ipix])*lwt
+                knb += 1
+
+            if knb == 0:
+                continue
+            ipeak = 0
+            p = nb_samples[0]
+            for isamp in range(1, get_num_samples(telid)):
+                if nb_samples[isamp] > p:
+                    p = nb_samples[isamp]
+                    ipeak = isamp
+            peakpos = peakpos_hg = ipeak
+            start = peakpos - parameters['nskip']
+
+            # Sanitity check?
+            if start < 0:
+                start = 0
+            if start + nsum > get_num_samples(telid):
+                start = get_num_samples(telid) - nsum
+
+            int_corr = set_integration_correction(telid, parameters)
+            # Integrate pixel
+            samples_pix_win = get_adc_sample(telid, igain)
+            [ipix][start:(nsum+start)]
+            ped_per_trace = ped[igain][ipix]/get_num_samples(telid)
+            sum_pix.append(round(int_corr[igain]*(sum(samples_pix_win) -
+                                                  ped_per_trace*nsum)))
+            peak_pix.append(peakpos)
+
+        sum_pix_tel.append(sum_pix)
+        time_pix_tel.append(peak_pix)
+
+    return sum_pix_tel, time_pix_tel
+
+
+def calibrate_amplitude_mc(integrated_charge, calib, telid, params):
+    TAG = sys._getframe().f_code.co_name+">"
+    """
+    Parameters
+    ----------
+    integrated_charge     Array of pixels with integrated change [ADC cts],
+                          pedestal substracted
+    calib                 Array of double containing the single-pe events
+    parameters['clip_amp']  Amplitude in p.e. above which the signal is
+                            clipped.
+    Returns
+    ------
+    Array of pixels with calibrate charge [photo-electrons]
+    Returns None if event is None
+
+    """
+
+    if integrated_charge is None:
+        return None
+    pe_pix_tel = []
+    for ipix in range(0, get_num_pixels(telid)):
+        pe_pix = 0
+        int_pix_hg = integrated_charge[get_num_channel(telid)-1][ipix]
+        # If the integral charge is between -300,2000 ADC ts, we choose the HG
+        # Otherwise the LG channel
+        # If there is only one gain, it is the HG (default)
+        if (int_pix_hg > -1000 and int_pix_hg < 10000 or
```

Comment:
this fails for me with an error "ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()."
Maybe you need  `(int_pix_hg >-1000).any()`?